language/Types.md

```diff
@@ -152,54 +161,88 @@ since the skolemized type variable ``a`` does not unify with ``Int``.
 
 ## Rows
 
-A row of types represents an unordered collection of named types without duplicates.
+A row of types represents an unordered collection of named types. Duplicate labels are allowed but are distinct.
```

Comment:
This?
> Rows are now unordered collections (of labels and types) with duplicates. However, the collection of types for a specific label within a row is ordered. Conceptually, a row can be thought of as a type-level Map Label (NonEmptyList Type).

What about ...
> A row of types represents an unordered collection of named types, with duplicates. Duplicate labels have their types collected together in order, as if in a ``NonEmptyList``. This means that, conceptually, a row can be thought of as a type-level ``Map Label (NonEmptyList Type)``.

Should we introduce the idea that labels can be strings too? (Just noticing that string labels are non-obvious but used in Halogen/DOM.)